en/deploy/README.md

```diff
@@ -131,6 +114,22 @@ $ git commit -m "My Django Girls app, first commit"
  create mode 100644 mysite/wsgi.py
 ```
 
+### Adjusting your branch name
```

Comment:
This section has the same content as before, with the exception of line 44 -> 127 where we need to change the reference to the next section. Before / after of the relevant change:

```diff
-[…] please continue to "Ignoring files".
+[…] please continue to "Pushing your code to GitHub".
```